DXRando/DeusEx/Classes/DXRandoCrowdControlEffects.uc

```diff
@@ -2239,6 +2247,90 @@ function bool RandomClothes()
 #endif
 }
 
+function int ShuffleBelt(String viewer)
+{
+    local int i,slot;
+    local #var(PlayerPawn) p;
+    local DeusExRootWindow root;
+    local Inventory beltItems[9],origBelt[9];
+    local Inventory invTemp;
+    local bool worked;
+
+    p = Player();
+
+    if (p==None){
+        return TempFail;
+    }
+
+    root = DeusExRootWindow(p.rootWindow);
+
+    if (root==None){
+        return TempFail;
+    }
+
+    //Items are in slots 1-9, Keyring is in 0
```

Comment:
...could we swap the keyring?

Reply:
It's possible, but since you can't move it yourself, it would be permanently locked in whatever location it gets shuffled too which would be a pretty nasty punishment.